lib/fizzy/parser.cpp

```diff
@@ -635,8 +636,8 @@ std::unique_ptr<const Module> parse(bytes_view input)
             if (export_.index >= total_global_count)
                 throw validation_error{"invalid index of an exported global"};
             break;
-        default:
-            assert(false);
+        default:                  // LCOV_EXCL_LINE
+            FIZZY_UNREACHABLE();  // LCOV_EXCL_LINE
```

Comment:
Are the LCOV lines needed if `fizzy::unreachable` has it?

Reply:
Yes. This is purely source based hint. The one for `return false` is needed because flush happens before and then the program crashes.